packages/cpt-ui/src/styles/card.scss

```diff
@@ -1,45 +1,61 @@
 .eps-card {
-  margin-bottom: 1.5rem;
-  padding: 0rem;
-
-  &__content {
-      display: flex;
-      padding: 0rem;
-      align-items: flex-start;
-  }
-
-  &__roleName {
-      margin-top: 0.5rem;
-      margin-bottom: 24px;
-      font-weight: normal;
-      font-size: 16px;
-      line-height: 1.5;
-      color: #4c6272;
-  }
-
-  &__siteAddress {
-      text-align: left;
-      color: #4c6272;
-      margin-top: 0.5rem;
-      font-size: 1rem;
-      line-height: 1.5;
-      &-line {
-          display: block;
-      }
-  }
-
-  &__link {
-    background-color: #ffffff;
-    border: 1px solid transparent;
-    box-shadow: 0 4px 0 0 #d8dde0;
-    display: block;
-    height: 100%;
-    position: relative;
-    text-decoration: none;
+    margin-bottom: 1.5rem;
+    padding: 0rem;
+
+    &__content {
+        display: flex;
+        padding: 0rem;
+        align-items: flex-start;
+    }
+
+    &__roleName {
+        margin-top: 0.5rem;
+        margin-bottom: 24px;
+        font-weight: normal;
+        font-size: 16px;
+        line-height: 1.5;
+        color: #4c6272;
+    }
+
+    &__siteAddress {
+        text-align: left;
+        color: #4c6272;
+        margin-top: 0.5rem;
+        font-size: 1rem;
+        line-height: 1.5;
+
+        &-line {
+            display: block;
+        }
+    }
+
+    &__link {
+        background-color: #ffffff;
+        border: 1px solid transparent;
+        box-shadow: 0 4px 0 0 #d8dde0;
+        display: block;
+        height: 100%;
+        position: relative;
+        text-decoration: none;
+    }
+
+    .nhsuk-grid-column-two-thirds {
```

Comment:
I moved this inside the eps-card class, to be sure that we're using it intentionally. I checked, and this move does not result in and layout changes to our pages. The uses of the `two-thirds` grid column are in the role selection page, and your selected role page, and there is no change after this move to either of them.